src/scripts/actions/VideoActions.js

```diff
@@ -26,14 +26,12 @@ export const fetchVideo = (id: string) => async (dispatch: Dispatch<*>) => {
   try {
     videoInfo = await paratii.core.vids.get(id)
 
-    console.log('videoInfo')
-    console.log(videoInfo)
-
     if (videoInfo) {
       videoInfo.id = videoInfo._id
     }
     if (videoInfo && videoInfo.id) {
-      dispatch(videoFetchSuccess(new VideoRecord(videoInfo)))
+      const { duration, ...videoInfoToSave } = videoInfo
```

Comment:
The logic here is basically to ignore the `duration` from the BE so that we can just rely on the number `clappr` gives us. I would like to remove this logic, but I think as long as `duration` is a string this will be a pain to deal with.